packages/contracts-bedrock/src/universal/OptimismMintableERC20Factory.sol

```diff
@@ -104,29 +109,29 @@ contract OptimismMintableERC20Factory is ISemver, Initializable {
     /// @param _name        ERC20 name.
     /// @param _symbol      ERC20 symbol.
     /// @param _decimals    ERC20 decimals
-    /// @return Address of the newly created token.
+    /// @return _localToken Address of the newly created token.
     function createOptimismMintableERC20WithDecimals(
         address _remoteToken,
         string memory _name,
         string memory _symbol,
         uint8 _decimals
     )
         public
-        returns (address)
+        returns (address _localToken)
```

Comment:
They use returns (address) instead of declaring the variable here

Reply:
Ok we can rollback it to make the less changes possible